tests/casestmt/tincompletecaseobject.nim

```diff
@@ -0,0 +1,45 @@
+discard """
```

Comment:
A smaller example based on a locally defined enum would be easier to read and would not require maintenance when there is a change in the `nnk*` enum values.